CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
+
+Starting with v0.21.0, TestCafe can serve proxied web pages over HTTPS. This allows you to test pages that require secure origin.
+
+To enable HTTPS when you use TestCafe through the command line, specify the [--ssl](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--ssl-options) flag followed by [HTTPS server options](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener). The most commonly used options are described in the [TLS topic](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options) in the Node.js documentation.
+
+```sh
+testcafe --ssl pfx=path/to/file.pfx;rejectUnauthorized=true;...
+```
+
+When you use programming API, pass HTTPS server options to the [createTestCafe](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/createtestcafe.html) method.
+
+```js
+'use strict';
+
+const createTestCafe        = require('testcafe');
+const selfSignedSertificate = require('openssl-self-signed-certificate');
+let runner                  = null;
+
+const sslOptions = {
+    key:  selfSignedSertificate.key,
+    cert: selfSignedSertificate.cert
+};
+
+createTestCafe('localhost', 1337, 1338, sslOptions)
+    .then(testcafe => {
+        runner = testcafe.createRunner();
+    })
+    .then(() => {
+        return runner
+            .src('test.js')
+
+            // Browsers restrict self-signed certificate usage unless you
+            // explicitly set a flag specific to each browser.
+            // For Chrome, this is '--allow-insecure-localhost'.
+            .browsers('chrome --allow-insecure-localhost')
+            .run();
+    });
+```
+
+See [Connect to TestCafe Server over HTTPS](https://devexpress.github.io/testcafe/documentation/using-testcafe/common-concepts/connect-to-the-testcafe-server-over-https.html) for more information.
+
+#### :gear: Construct Screenshot Paths with Patterns ([#2152](https://github.com/DevExpress/testcafe/issues/2152))
+
+You can now use patterns to construct paths to screenshots. TestCafe provides a number of variables you can include in the path. For instance, these are `${DATE}`, `${TIME}`, `${USERAGENT}`, etc. For a complete list, refer to the command line [--screenshot-path-pattern flag description](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-p---screenshot-path-pattern).
+
+You specify a screenshot path pattern when you run tests. Each time TestCafe takes a screenshot, it substitutes the variables with their values and saves the screenshot to the resulting path.
```

Comment:
This phrase is not clear.
Something like this `..it substitutes the placeholders with actual values and saves the screenshot to ...`